SonarQube.MSBuild.Tasks/Targets/SonarQube.Integration.targets

```diff
@@ -114,7 +114,7 @@
   <!-- **************************************************************************** -->
   <!-- **************************************************************************** -->
 
-  <Target Name="SkippingSonarQubeAnalysis" BeforeTargets="CoreCompile"
+  <Target Name="SkippingSonarQubeAnalysis" BeforeTargets="Build"
           Condition=" $(SonarQubeTempPath) == '' ">
```

Comment:
CoreCompile only applies to managed targets (the C++ equivalent is ClCompile).
However, both managed and C++ have a Build target, and indeed MSBuild looks for and executes the Build target target by default unless you specifically tell it to execute something else.

It doesn't matter much when the "SkippingSonarQubeAnalysis" target is executed as it only produces diagnostic info.